scripts/remove.sh

```diff
@@ -463,6 +463,36 @@ remove_MoonrakerTelegramBot(){
   CONFIRM_MSG="MoonrakerTelegramBot successfully removed!"
 }
 
+remove_MoonCord(){
+  source_kiauh_ini
```

Comment:
I don't think this is needed in this function? It seems you don't use anything from the kiauh.ini in this function.
So this line can get removed completely i guess.